tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/AspNetCore5IastTests.cs

```diff
@@ -162,6 +162,53 @@ await VerifyHelper.VerifySpans(spans, settings)
                               .DisableRequireUniquePrefix();
         }
 
+        [SkippableFact]
+        [Trait("Category", "ArmUnsupported")]
```

Comment:
Why ArmUnsupported? 🤔

Reply:
It was wrongly inherited from other tests. I have removed them.